### ISSUE TITLE
Missing mail class, view file for invoice reminders

### ISSUE THREAD

**Issue by ghermans** [bagisto/bagisto#10537]
### Description

The sendInvoiceReminder() method in the **InvoiceReminder** trait is calling 
`Mail::queue(new InvoiceOverdueReminder($customer, $this));` 
but the InvoiceOverdueReminder mail class, and its corresponding view file are missing.

### Describe the solution you'd like

1. Create the InvoiceOverdueReminder mail class.
2. Create a email view file.
3. Add translations in the respective language files

**Comment by devansh-webkul** [comment]
Mail class is missing and mail view is missing which needs to be develop for the new theme. Marking this as a bug.
